app/helpers/application_helper/toolbar/physical_racks_center.rb

```diff
@@ -1,2 +1,23 @@
 class ApplicationHelper::Toolbar::PhysicalRacksCenter < ApplicationHelper::Toolbar::Basic
+  button_group('ems_physical_rack_vmdb',
+               [
+                 select(
+                   :physical_rack_vmdb_choice,
+                   'fa fa-cog fa-lg',
+                   t = N_('Configuration'),
+                   t,
+                   :items => [
+                     button(
+                       :physical_rack_refresh,
+                       'fa fa-refresh fa-lg',
+                       N_('Refresh relationships and power states for all items related to this Infrastructure Provider'),
+                       N_('Refresh Relationships and Power States'),
+                       :data    => {'function'      => 'sendDataWithRx',
+                                    'function-data' => '{"type": "refresh", "controller": "toolbarActions", "payload": {"entity": "physical_racks"}}'},
+                       :confirm => N_("Refresh relationships and power states for all items related to selected Physical Racks?"),
+                       :options => {:feature => :refresh}
```

Comment:
This is probably missing an `onwhen` attribute, so that this isn't enabled for 0 items.